bin/node/runtime/Cargo.toml

```diff
@@ -132,3 +132,4 @@ std = [
 	"pallet-recovery/std",
 	"pallet-vesting/std",
 ]
+runtime_benchmarks = ["frame-benchmarking"]
```

Comment:
For consistency, I'd prefer this feature to be called `runtime-benchmarks`.

Reply:
Makes sense.